### DISCUSSION TITLE
Manim v0.6.0

### DISCUSSION THREAD

**Discussion by behackl** [ManimCommunity/manim#1442] [Announcements]
The full changelog can be found at <https://docs.manim.community/en/latest/changelog/0.6.0-changelog.html>.

<hr /><em>This discussion was created from the release <a href='https://github.com/ManimCommunity/manim/releases/tag/v0.6.0'>Manim v0.6.0</a>.</em>

**Comment by kilacoda-old** [comment]
Should this be renamed to `v0.6.0` instead of `Manim v0.6.0` to keep the format of previous releases?

**Reply by behackl** [reply]
I think the GitHub release name doesn't matter too much; @naveen521kk suggested a more verbose name (which I liked). 🤷

**Reply by naveen521kk** [reply]
It was just a personal preference though.
